back/app/models/home_page.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# Contains settings & configuration data of the homepage.
```

Comment:
I think `projects_header` should be renamed to `projects_header_multiloc`

Reply:
Good idea.

Reply:
Thanks for doing this Simon. :+1: